krakentools/d4a2fbe/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:jammy as app
+
+WORKDIR /
+
+# using a git commit because the last release was over three years ago,, despite recent(ish) activity in the repo's main branch
+ARG KRAKENTOOLS_VERSION="d4a2fbe2833c2392a724697c0c97a3014e748474"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="KrakenTools"
+LABEL software.version=${KRAKENTOOLS_VERSION}
+LABEL description="KrakenTools is a set of tools for processing Kraken output"
+LABEL website="https://github.com/jenniferlu717/KrakenTools"
+LABEL documentation="https://github.com/jenniferlu717/KrakenTools"
+LABEL license="https://github.com/jenniferlu717/KrakenTools/blob/master/LICENSE"
+LABEL maintainer="Sage Wright"
+LABEL maintainer.email="sagemwright@gmail.com"
+
+
+# Install dependencies via apt-get; cleanup apt garbage
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget \
+    ca-certificates \
+    python3 \
+    python3-pip \
+    python3-setuptools \
+    gawk \
+    jq \
```

Comment:
Would you like to include/keep gawk and jq? I don't see them listed as dependencies but understand if you'd like those in the docker image anyways